### DISCUSSION TITLE
Fixing ISP blocks

### DISCUSSION THREAD

**Discussion by Zonalds** [vercel/community#7025] [Help] [answered]
### Question

I noticed major ISPs in Nigeria are blocking Vercel sites. Not sure why but without a VPN it doesn't work. 

My question is; is there a way to fix this even if it means upgrading?

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out. Upgrading is not necessary and will not solve the problem. We are in contact with the Nigerian government and ISPs to remove the block block against our 76.76.21.21 IP address.

A few people have reported a workaround in https://github.com/vercel/community/discussions/3630#discussioncomment-9482248. I've also seen [Vercel nameservers](https://vercel.com/guides/zero-downtime-migration-for-dns) or use of an ALIAS record proposed as possible workarounds. If you try any of those options and find a strategy that works for your site, you can let the community know about it in https://github.com/vercel/community/discussions/3630 🙇
